ip_address: "192.168.1.250"
sensor_x: 0.0
sensor_y: 0.0
sensor_z: 0.07
sensor_rx: 0.0
sensor_ry: 0.0
sensor_rz: 0.0
min_x: -35.0
max_x: 35.0
min_y: -35.0
max_y: 35.0
min_z: 0.0
max_z: 5.0
min_range: 0.0
max_range: 35.0
min_vertical_angle: -40.0
max_vertical_angle: 40.0
min_horizontal_angle: -180.0
max_horizontal_angle: 180.0
noise_filter_level: 30.0
